builder/metadata.go

```diff
@@ -187,18 +198,20 @@ func (b *Builder) UpdateVersions(nextMix, nextUpstream uint32) error {
 
 	nextUpstreamStr := strconv.FormatUint(uint64(nextUpstream), 10)
 
-	nextFormat := format
-	if nextUpstream > latest {
-		nextFormat, err = b.getUpstreamFormat(nextUpstreamStr)
-		if err != nil {
-			return err
+	if checkUpstream {
+		nextFormat = format
```

Comment:
`nextFormat` is only set if you are going to `checkUpstream` so it will be printed (on line 217 for now) as its zero-value. You need to initialize it to the current format.

Reply:
That's invalid though, the right answer is to not print it at all because the nextFormat is not guaranteed to be curr +1.

Reply:
Updated to check if it's empty.

Reply:
No, next format is guaranteed to be current format if you aren't changing your upstream, why not display that? You are still printing what the `nextUpstream` is even if it didn't change but for some reason you are deciding not to print what the format of that upstream is? So I'll say again (feel free to give me a thumbs down again if you wish): you should initialize it to the current format and always print.

Reply:
It doesn't make sense to me to say the next format = my current format, that's why there's 2 format variables to print here. If I can't check for format and my curr format is 25, my next format is not 25, it's (likely) 26, but who knows, it could be 30. I would rather not print information than print misleading info.

Reply:
but you are just printing what the format the next upstream version is. If the version itself hasn't changed why would the upstream format change?

Reply:
Ok I think we were not on the same page, I think I've implemented what you're stating above now ^